<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>CalibrationWidget</class>
 <widget class="QWidget" name="CalibrationWidget">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1306</width>
    <height>738</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Kalibrierungstool</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout_4">
   <item>
    <widget class="QSplitter" name="splitter">
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
     <widget class="QWidget" name="verticalLayoutWidget">
      <layout class="QVBoxLayout" name="verticalLayout_15">
       <item>
        <layout class="QHBoxLayout" name="verticalLayout_16">
         <item>
          <widget class="QLabel" name="label_13">
           <property name="text">
            <string>Ansicht:</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QComboBox" name="comboBox_ansicht">
           <property name="sizePolicy">
            <sizepolicy hsizetype="Preferred" vsizetype="Fixed">
             <horstretch>0</horstretch>
             <verstretch>0</verstretch>
            </sizepolicy>
           </property>
           <property name="minimumSize">
            <size>
             <width>230</width>
             <height>0</height>
            </size>
           </property>
           <item>
            <property name="text">
             <string>Ausgangsbild</string>
            </property>
           </item>
           <item>
            <property name="text">
             <string>Entzerrtes Bild</string>
            </property>
           </item>
           <item>
            <property name="text">
             <string>Gefundene Schachbrettecken</string>
            </property>
           </item>
          </widget>
         </item>
         <item>
          <spacer name="horizontalSpacer">
           <property name="orientation">
            <enum>Qt::Horizontal</enum>
           </property>
           <property name="sizeHint" stdset="0">
            <size>
             <width>40</width>
             <height>20</height>
            </size>
           </property>
          </spacer>
         </item>
        </layout>
       </item>
       <item>
        <widget class="ResizeableGraphicsView" name="graphicsView">
         <property name="minimumSize">
          <size>
           <width>640</width>
           <height>480</height>
          </size>
         </property>
         <property name="baseSize">
          <size>
           <width>640</width>
           <height>480</height>
          </size>
         </property>
        </widget>
       </item>
      </layout>
     </widget>
     <widget class="QWidget" name="layoutWidget">
      <layout class="QVBoxLayout" name="verticalLayout_3">
       <item>
        <widget class="QTabWidget" name="tabWidget">
         <property name="currentIndex">
          <number>1</number>
         </property>
         <widget class="QWidget" name="tab">
          <attribute name="title">
           <string>Kalibrierbilder</string>
          </attribute>
          <layout class="QVBoxLayout" name="verticalLayout_2">
           <item>
            <widget class="QLabel" name="label">
             <property name="text">
              <string>Kalibrierbilder:</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QTableView" name="tableView_images"/>
           </item>
           <item>
            <layout class="QVBoxLayout" name="verticalLayout">
             <item>
              <layout class="QHBoxLayout" name="horizontalLayout">
               <item>
                <widget class="QPushButton" name="pushButton_loeschen">
                 <property name="text">
                  <string>Entfernen</string>
                 </property>
                </widget>
               </item>
               <item>
                <widget class="QPushButton" name="pushButton_hinzufuegen">
                 <property name="text">
                  <string>Hinzufügen</string>
                 </property>
                </widget>
               </item>
              </layout>
             </item>
             <item>
              <widget class="QPushButton" name="pushButton_ordnerHinzufuegen">
               <property name="text">
                <string>Ordner hinzufügen</string>
               </property>
              </widget>
             </item>
            </layout>
           </item>
          </layout>
         </widget>
         <widget class="QWidget" name="tab_2">
          <attribute name="title">
           <string>Einstellungen</string>
          </attribute>
          <layout class="QVBoxLayout" name="verticalLayout_6">
           <item>
            <layout class="QVBoxLayout" name="verticalLayout_5">
             <item>
              <widget class="QLabel" name="label_7">
               <property name="text">
                <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;&lt;span style=&quot; font-weight:600;&quot;&gt;Kalibriermuster:&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
               </property>
              </widget>
             </item>
             <item>
              <layout class="QHBoxLayout" name="horizontalLayout_6">
               <item>
                <widget class="QLabel" name="label_8">
                 <property name="text">
                  <string>Quadratgröße:</string>
                 </property>
                </widget>
               </item>
               <item>
                <widget class="QLineEdit" name="lineEdit_quadratGroesse">
                 <property name="text">
                  <string>0.0068</string>
                 </property>
                </widget>
               </item>
              </layout>
             </item>
             <item>
              <layout class="QHBoxLayout" name="horizontalLayout_5">
               <item>
                <widget class="QLabel" name="label_9">
                 <property name="text">
                  <string>Ecken horizontal:</string>
                 </property>
                </widget>
               </item>
               <item>
                <widget class="QLineEdit" name="lineEdit_eckenHorizontal">
                 <property name="text">
                  <string>8</string>
                 </property>
                </widget>
               </item>
              </layout>
             </item>
             <item>
              <layout class="QHBoxLayout" name="horizontalLayout_4">
               <item>
                <widget class="QLabel" name="label_10">
                 <property name="text">
                  <string>Ecken vertikal:</string>
                 </property>
                </widget>
               </item>
               <item>
                <widget class="QLineEdit" name="lineEdit_eckenVertikal">
                 <property name="text">
                  <string>6</string>
                 </property>
                </widget>
               </item>
              </layout>
             </item>
            </layout>
           </item>
           <item>
            <spacer name="verticalSpacer">
             <property name="orientation">
              <enum>Qt::Vertical</enum>
             </property>
             <property name="sizeHint" stdset="0">
              <size>
               <width>20</width>
               <height>40</height>
              </size>
             </property>
            </spacer>
           </item>
          </layout>
         </widget>
         <widget class="QWidget" name="tab_3">
          <attribute name="title">
           <string>Ergebnisse</string>
          </attribute>
          <layout class="QVBoxLayout" name="verticalLayout_12">
           <item>
            <layout class="QVBoxLayout" name="verticalLayout_11">
             <item>
              <widget class="QLabel" name="label_11">
               <property name="text">
                <string>Kameramatrix:</string>
               </property>
              </widget>
             </item>
             <item>
              <widget class="QLabel" name="label_cameraMatrix">
               <property name="text">
                <string>&lt;!DOCTYPE html PUBLIC &quot;-//W3C//DTD HTML 4.01//EN&quot;&gt;

&lt;html&gt;
&lt;head&gt;

&lt;body&gt;
    &lt;table border=&quot;0&quot; cellpadding=&quot;2&quot; cellspacing=&quot;2&quot; style=
    &quot;margin-top:0px; margin-bottom:0px; margin-left:0px; margin-right:0px;&quot;&gt;
        &lt;tr&gt;
            &lt;td&gt;
                &lt;p&gt;&lt;span style=&quot;font-size:12pt;&quot;&gt;1.0&lt;/span&gt;&lt;/p&gt;
            &lt;/td&gt;

            &lt;td&gt;
                &lt;p&gt;&lt;span style=&quot;font-size:12pt;&quot;&gt;0.0&lt;/span&gt;&lt;/p&gt;
            &lt;/td&gt;

            &lt;td&gt;
                &lt;p&gt;&lt;span style=&quot;font-size:12pt;&quot;&gt;0.0&lt;/span&gt;&lt;/p&gt;
            &lt;/td&gt;
        &lt;/tr&gt;

        &lt;tr&gt;
            &lt;td&gt;
                &lt;p&gt;&lt;span style=&quot;font-size:12pt;&quot;&gt;0.0&lt;/span&gt;&lt;/p&gt;
            &lt;/td&gt;

            &lt;td&gt;
                &lt;p&gt;&lt;span style=&quot; font-size:12pt;&quot;&gt;1.0&lt;/span&gt;&lt;/p&gt;
            &lt;/td&gt;

            &lt;td&gt;
                &lt;p&gt;&lt;span style=&quot; font-size:12pt;&quot;&gt;0.0&lt;/span&gt;&lt;/p&gt;
            &lt;/td&gt;
        &lt;/tr&gt;

        &lt;tr&gt;
            &lt;td&gt;
                &lt;p&gt;&lt;span style=&quot; font-size:12pt;&quot;&gt;0.0&lt;/span&gt;&lt;/p&gt;
            &lt;/td&gt;

            &lt;td&gt;
                &lt;p&gt;&lt;span style=&quot; font-size:12pt;&quot;&gt;0.0&lt;/span&gt;&lt;/p&gt;
            &lt;/td&gt;

            &lt;td&gt;
                &lt;p&gt;&lt;span style=&quot; font-size:12pt;&quot;&gt;1.0&lt;/span&gt;&lt;/p&gt;
            &lt;/td&gt;
        &lt;/tr&gt;
    &lt;/table&gt;
&lt;/body&gt;
&lt;/html&gt;</string>
               </property>
              </widget>
             </item>
            </layout>
           </item>
           <item>
            <layout class="QVBoxLayout" name="verticalLayout_10">
             <item>
              <widget class="QLabel" name="label_14">
               <property name="text">
                <string>Verzerrungskoeffizienten:</string>
               </property>
              </widget>
             </item>
             <item>
              <widget class="QLabel" name="label_distoritionCoefficents">
               <property name="text">
                <string>[0.0,0.0,0.0]</string>
               </property>
              </widget>
             </item>
            </layout>
           </item>
           <item>
            <layout class="QVBoxLayout" name="verticalLayout_9">
             <item>
              <widget class="QLabel" name="label_12">
               <property name="text">
                <string>Rückrpojektionsfehler:</string>
               </property>
              </widget>
             </item>
             <item>
              <widget class="QLabel" name="label_reprojectionError">
               <property name="text">
                <string>0.0</string>
               </property>
              </widget>
             </item>
            </layout>
           </item>
           <item>
            <spacer name="verticalSpacer_2">
             <property name="orientation">
              <enum>Qt::Vertical</enum>
             </property>
             <property name="sizeHint" stdset="0">
              <size>
               <width>20</width>
               <height>40</height>
              </size>
             </property>
            </spacer>
           </item>
          </layout>
         </widget>
        </widget>
       </item>
       <item>
        <layout class="QHBoxLayout" name="horizontalLayout_7">
         <item>
          <widget class="QPushButton" name="pushButton_kalibrierdatenLaden">
           <property name="text">
            <string>Kalibrierdaten laden</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QPushButton" name="pushButton_kalibrieren">
           <property name="text">
            <string>Kalibrieren</string>
           </property>
          </widget>
         </item>
        </layout>
       </item>
      </layout>
     </widget>
    </widget>
   </item>
   <item>
    <widget class="QProgressBar" name="progressBar">
     <property name="value">
      <number>0</number>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <customwidgets>
  <customwidget>
   <class>ResizeableGraphicsView</class>
   <extends>QGraphicsView</extends>
   <header>ResizeableGraphicsView.h</header>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
